manifests/i/Insomnia/Insomnia/core@2021.7.1/Insomnia.Insomnia.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Insomnia.Insomnia
+PackageVersion: core@2021.7.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.1/Insomnia.Core-2021.7.1-portable.exe
```

```suggestion
  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.1/Insomnia.Core-2021.7.1.exe
```

You need to fix the matching